docs/checks.md

```diff
@@ -97,6 +97,28 @@ Additionally, in some cases these rules will need to be suspended. For example,
 if a past commit includes illegal content such as child pornography, it may be
 necessary to use a force push to rewrite the history rather than simply hide the
 commit. 
+
+This test has tiered scoring. Each tier must be fully satisfied to achieve points at the next tier. For example, if you fulfill the Tier 3 checks but do not fulfill all the Tier 2 checks, you will not receive any points for Tier 3.
+
+Note: If Scorecard is run without an administrative access token, the requirements that specify “For administrators” are ignored.
+
+Tier 1 Requirements (3/10 points):
+  - Force push
```

Comment:
suggest:
- Prevent force push
- Prevent branch deletion
- Include administrator for review

Reply:
done, thanks.